Makefile

```
@@ -36,7 +36,7 @@ info:
 	$(info usage: `make MEASURE_DIR=/path/to/measure/dir VERSION=x.y.z)
 
 .check-dependencies:
-	npm install -g fhir-bundle-calculator
+	#npm install -g fhir-bundle-calculator
```

```suggestion
	npm install -g fhir-bundle-calculator
```
Should uncomment this. This probably was me that commented it out. Sorry.